iotdb-core/datanode/src/main/java/org/apache/iotdb/db/queryengine/plan/planner/plan/node/load/LoadSingleTsFileNode.java

```diff
@@ -151,7 +152,7 @@ public TRegionReplicaSet getRegionReplicaSet() {
 
   @Override
   public List<PlanNode> getChildren() {
-    return null;
+    return Collections.emptyList();
```

Comment:
I found that if this function returns null here:
- In Windows, it returns null.
- In Windows Server and Unix, it returns an empty list.

So I made it return an empty list explicitly.